### ISSUE TITLE
The products dataset could not be downloaded successfully

### ISSUE THREAD

**Issue by xuziweiwh** [a1bc2def6g/fastgl-ae#1]
Hello, when I run the run_fig9_fastgl.sh command, the reddit dataset can be downloaded successfully, but when the products dataset is downloaded is very slow (only half an hour to 10%), and at the end it shows that the download failed. Later, I successfully downloaded the dataset via the products dataset download URL that appeared at runtime, but then it was missing some of the files that appeared when the dataset was generated by code run. For example, if I successfully download a reddit dataset, a reddit_69f818f5 folder will be generated in the dataset file, is there any way to speed up the download?  Looking forward to hearing from you!
Environment configuration：
Ubuntu: 22.04
CUDA: 12.4
Python: 3.9
Pytorch: 2.4.0
Torch Geometric: 2.6.1
DGL: 2.4.0+cu124
OGB: 1.3.6
Numpy: 1.26.4
Scikit Learn : 1.5.2
Matplotlib: 3.3.4

**Comment by weihai-98** [comment]
Yes, I also download the datasets via url in the browser through some vpn, which may speed up the download process. Then you can unzip the file to a folder and process it with the corresponding function of dgl or pyg. products dataset after downloading is the file structure as shown below. Hope this helps.
![products](https://github.com/user-attachments/assets/16bb676e-b089-48b3-9c63-ac0b9dbb3fb6)

**Comment by xuziweiwh** [comment]
Hello, how can I use the dgl or pyg function to generate the following format for the products dataset?
![数据集](https://github.com/user-attachments/assets/4b905fa5-a1fd-4e33-891a-67da8ebdcf40)

**Comment by weihai-98** [comment]
Please directly unzip the zip file and process the raw file with the function provided by pyg or dgl, e.g., `from dgl.data import RedditDataset` for reddit, `AsNodePredDataset(DglNodePropPredDataset(args.dataset_name, root=path_to_folder))`  for products.

**Comment by xuziweiwh** [comment]
Hello, because the host GPU is not enough, I re-ran this code with the server, and I am still having trouble downloading the dataset. I ran the ./run_fig9_fastdgl.sh command, and the first three datasets were successfully downloaded, but the mag dataset and igb-large dataset were not downloaded (resulting in me reporting mag_coarse.bin and igb_large.bin not existing). Can you tell me how to download these two?
